### PR TITLE
Make some missed fields private and/or final

### PR DESCRIPTION
**Pull request by pkubowicz**
Cleans up 36d08fd1fd82598636f04

### CODE REVIEW

File: simpleclient/src/main/java/io/prometheus/client/Collector.java [hunk 1]
```diff
@@ -140,9 +140,9 @@ public String toString() {
    */
   public static final double MILLISECONDS_PER_SECOND = 1E3;
 
-  protected static final Pattern METRIC_NAME_RE = Pattern.compile("[a-zA-Z_:][a-zA-Z0-9_:]*");
-  protected static final Pattern METRIC_LABEL_NAME_RE = Pattern.compile("[a-zA-Z_][a-zA-Z0-9_]*");
-  protected static final Pattern RESERVED_METRIC_LABEL_NAME_RE = Pattern.compile("__.*");
+  private static final Pattern METRIC_NAME_RE = Pattern.compile("[a-zA-Z_:][a-zA-Z0-9_:]*");
```
Comment (brian-brazil) [inline simpleclient/src/main/java/io/prometheus/client/Collector.java]: These should probably be public.

Comment (pkubowicz) [inline simpleclient/src/main/java/io/prometheus/client/Collector.java]: You have protected-level methods to perform those checks, you don't need access to the regexes.

Comment (brian-brazil) [inline simpleclient/src/main/java/io/prometheus/client/Collector.java]: Ah right, it's been a while since I've wrote this code.

**Comment by brian-brazil** [comment]
Thanks!

**Pull request merged**